include/dlr_internal.h

```diff
@@ -0,0 +1,123 @@
+#ifndef DLR_INTERNAL_H_
+#define DLR_INTERNAL_H_
+
+#include <graph/graph_runtime.h>
+#include <tvm/runtime/module.h>
+#include <treelite/predictor.h>
+#include <treelite/c_api_runtime.h>
+#include "../3rdparty/treelite/dmlc-core/src/io/filesys.h"
+//#include <graph/graph_runtime.cc>
+//#include <graph/debug/graph_runtime_debug.cc>
+#include <runtime_base.h>
+#include <dlpack/dlpack.h>
+#include <string>
+#include <vector>
+#include <sys/types.h>
+#include "dlr.h"
+
+#define LINE_SIZE 256
+
+
+typedef struct {
+  std::string model_lib;
+  std::string params;
+  std::string model_json;
+  std::string ver_json;
+} ModelPath;
+
+void listdir(const std::string& dirname, std::vector<std::string> &paths);
+
+inline bool endsWith(const std::string &mainStr, const std::string &toMatch)
+{
+	if(mainStr.size() >= toMatch.size() &&
+	  mainStr.compare(mainStr.size() - toMatch.size(), toMatch.size(), toMatch) == 0)
+		return true;
+	else
+		return false;
+}
+
+enum class DLRBackend {
+  kTVM,
+  kTREELITE
+};
+/*! /brief Get the paths of the TVM model files.
+ */
+ModelPath get_tvm_paths(const std::string &tar_path);
```

Comment:
Let's use the left pointer alignment style (`const std::string& tar_path`) to keep the consistency.

Reply:
There are 9 places in DLR `.cc` and `.h` files where we use `string &<var>` and 100+ places in `3rdparty` where they use `string &<var>`.
Lets discuss it.
Common recommendation is to put `*` in pointer declarations to the variable
https://stackoverflow.com/questions/180401/placement-of-the-asterisk-in-pointer-declarations
```
int *pointer1, *pointer2; // Fully consistent, two pointers
int* pointer1, pointer2;  // Inconsistent (because only the first one is a pointer, the second one is an int variable), unexpected, and thus prone to errors
```

Reply:
Agreed, I think we should use left alignment and avoid all multi-decl in the same line. 

https://dev.to/codemouse92/cc-pointer-alignment-style-a-justification-13po